functions-methods.js

```diff
@@ -33,4 +58,19 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(email) {
+    const atCheck = email.includes("@");
+    const commaCheck = email.includes(",");
+    const dotCheck = email.charAt(email.length - 1) == ".";
+
+    if ((atCheck == true) &&  (commaCheck == false) && (dotCheck == false)) {
+        return true;
+    }
+    else {
+        return false;
+    }
```

Comment:
Je kan de uitkomst van de expressie meteen teruggeven:
```suggestion
    return atCheck && !commaCheck && !dotCheck;
```